src/lib/__snapshots__/create-config.test.ts.snap

```diff
@@ -20,10 +20,8 @@ exports[`should create default config 1`] = `
     "demoAllowAdminLogin": false,
     "enableApiToken": true,
     "initApiTokens": [],
-    "initialAdminUser": {
-      "password": "unleash4all",
-      "username": "admin",
-    },
+    "initialAdminEmailUser": undefined,
```

Comment:
Isn't this a breaking change? This is part of the public config isn't it?

Reply:
Yes, however the type was already optional before, which was a lie anyways. I just removed the defaults since they were not being used anyways and actually handled this as an optional.